AirCasting/SDSync/Models/SDStream.swift

```diff
@@ -10,3 +10,8 @@ struct SDStream: Hashable {
     let header: SDCardCSVFileFactory.Header
 }
 
+extension String {
```

Comment:
hm, I'd probably go for a class SessionUUID that is an extension of string and has this property, since it doesn't make sense for ALL the strings.

Reply:
I'm not sure I understand. 
SessionUUID is a different thing than deviceID. I could change it to extension of `AirBeamDeviceType` but it requires some changes that would require another CR probably. So I'm skipping this for now and it's to be discussed, ok? :)